lib/StaticNarrative/narrative_ref.py

```diff
@@ -9,17 +9,18 @@
 
 class NarrativeRef:
     def _less_than_zero(self: "NarrativeRef", number: str | int, name: str) -> int:
+        err_msg = f"The Narrative {name} must be an integer > 0, not {number}"
         try:
             integer = int(number)
-            if integer <= 0:
-                raise ValueError
-        except (ValueError, TypeError) as e:
-            err = f"The Narrative {name} must be an integer > 0, not {number}"
-            raise ValueError(err) from e
+        except (TypeError, ValueError) as e:
+            raise ValueError(err_msg) from e
```

Comment:
ruff didn't like the raising of the ValueError, which was then caught and turned into another value error.